jwt-auth.php

```diff
@@ -0,0 +1,72 @@
+<?php
+/**
+ * JWT Auth
+ *
+ * @package      JWTAuth
+ * @author       XWP
+ * @license      GPL-2.0+
+ *
+ * @wordpress-plugin
+ * Plugin Name:       JWT Auth
+ * Plugin URI:        https://github.com/WP-API/jwt-auth
+ * Description:       Feature plugin to bring JSON Web Token REST API authentication to Core
+ * Version:           0.1
+ * Author:            XWP, and contributors
+ * Author URI:        https://github.com/WP-API/jwt-auth/graphs/contributors
+ * Text Domain:       jwt-auth
+ * License:           GPL-2.0+
+ * License URI:       http://www.gnu.org/licenses/gpl-2.0.txt
+ * GitHub Plugin URI: https://github.com/WP-API/jwt-auth
+ * Requires PHP:      5.3.0
+ * Requires WP:       4.4.0
+ */
+
+define( 'JWT_AUTH_PLUGIN_DIR', dirname( __FILE__ ) );
+define( 'JWT_AUTH_PLUGIN_URL', plugin_dir_url( __FILE__ ) );
+define( 'JWT_AUTH_VERSION', '0.1' );
+
+/**
+ * Requires running PHP 5.3 or above.
+ *
+ * @since 0.1
+ * @codeCoverageIgnore
+ */
+function jwt_auth_version_check() {
+
+	if ( version_compare( PHP_VERSION, '5.3', '<' ) ) {
+		deactivate_plugins( plugin_basename( __FILE__ ) );
+		wp_die( esc_html__( 'The JWT Auth plugin requires PHP Version 5.3 or above.', 'jwt-auth' ) );
```

Comment:
See above note about 5.6

Reply:
@kadamwhite Changing this to 5.6 is not needed. The minimum PHP version required for the code to work is 5.3 and Core is adopting 5.3, so what would the benefit of making this 5.6 be?

Reply:
Per this post: https://wordpress.org/news/2019/04/minimum-php-version-update/ the PHP minimum version for core is, "5.6 now and soon 7+"